.github/workflows/continuous-integration.yaml

```
@@ -10,6 +10,42 @@ on:
       - master
 
 jobs:
+  fix-formatting:
+    if: >-
+      github.actor != 'github-actions[bot]' &&
+      github.event_name == 'pull_request' &&
+      github.pull_request.head.repo.fork == true
```

Did you mean `false`? The action will not be able to push to the fork

```suggestion
      github.pull_request.head.repo.fork == false
```

---

Yes, I did mean false. Thanks.